homeassistant/components/switchbot/binary_sensor.py

```diff
@@ -0,0 +1,77 @@
+"""Support for SwitchBot binary sensors."""
+from __future__ import annotations
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_MAC, CONF_NAME
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DATA_COORDINATOR, DOMAIN
+from .coordinator import SwitchbotDataUpdateCoordinator
+from .entity import SwitchbotEntity
+
+PARALLEL_UPDATES = 1
+
+BINARY_SENSOR_TYPES: dict[str, BinarySensorEntityDescription] = {
+    "calibration": BinarySensorEntityDescription(
+        key="calibration",
+        device_class=None,
```

Comment:
You should be able to drop any of the keys with a value of none

Reply:
Tested this and it works fine without the none keys.  Dropped none keys as suggested.